README.md

```diff
@@ -344,6 +356,19 @@ vars:
 * `variable` (required): The variable containing the calculated field `name` and `transform_sql`.
 
 ----
+
+### drop_schemas ([source](macros/drop_schemas.sql))
```

Comment:
I would maybe give an examples of how it can be used. 

I can see two use cases --
1) at the end of buildkite runs like you mentioned
2) in the deployments.yml

Reply:
I second this. It will be helpful to understand how this is being used in practice.

Reply:
added!